Source/WebKit/UIProcess/ios/WKTouchEventsGestureRecognizer.mm

```diff
@@ -227,7 +228,40 @@ - (void)_updateTapStateWithTouches:(NSSet *)touches type:(WebKit::WKTouchEventTy
     }
 }
 
-- (void)_recordTouches:(NSSet<UITouch *> *)touches type:(WebKit::WKTouchEventType)type
+- (WebKit::WKTouchEvent)_coalescedTouchEventForTouch:(UITouch *)touch
```

Comment:
Isn't this just `touchEventFor(UI)Touch`? Since this method doesn't actually deal with any coalescing. We could give it any `UITouch *` instance and it would be happy.

Reply:
yes; I will change this in a follow up PR